transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/shyrka/receive/StructuredMessage.kt

```diff
@@ -110,6 +113,32 @@ internal data class StructuredMessage(
             )
         }
 
+        @Serializable
+        data class CardContent(
+            val contentType: String,
+            val card: Card,
+        ) : Content() {
+            @Serializable
+            data class Card(
+                val title: String,
```

Comment:
Are all fields in `Card` mandatory? Do we know if it is possible to create a Card without description,title or image? I think, I remember we said that image can be optional. (but not 100$ sure)

Reply:
Just want to confirm. The `text` and `description` are mandatory?